astropy/cosmology/tests/test_cosmology.py

```diff
@@ -1577,6 +1577,21 @@ def test_z_at_value():
     assert allclose(z_at_value(cosmo.distmod, 46 * u.mag),
                     1.9913891680278133, rtol=1e-6)
 
+    # Test array:
+    assert allclose(z_at_value(cosmo.distmod, np.array([46]*4) * u.mag,
```

Comment:
Nitpick: just NumPy has a func for making full arrays.

```suggestion
    assert allclose(z_at_value(cosmo.distmod, np.full(4,46) * u.mag,
```